RNTester/js/utils/RNTesterList.ios.js

```diff
@@ -249,6 +249,12 @@ const APIExamples: Array<RNTesterExample> = [
     module: require('../examples/Alert/AlertIOSExample'),
     supportsTVOS: true,
   },
+  // [TODO(macOS ISS#2323203)
+  {
+    key: 'AlertMacOSExample',
+    module: require('../examples/Alert/AlertMacOSExample'),
+    supportsTVOS: true,
```

Comment:
Is this true?

Reply:
ha.  That's what it was before it was removed in the .61 merge.   This will go away again after issue #354.